src/test/java/org/junit/tests/assertion/MultipleFailureExceptionTest.java

```diff
@@ -67,6 +68,16 @@ public void assertEmptyThrowsMutipleFailureExceptionForManyThrowables() throws E
         }
     }
 
+    @Test
+    public void assertEmptyErrorListConstructorFailure() throws Exception {
+        try {
+            new MultipleFailureException(Collections.<Throwable>emptyList());
+            fail();
```

Comment:
I will catch `IllegalArgumentException` instead of  `Exception`, although 'AssertionError' is-not-a `Exception` :)